examples/traffic-light.manifold

```diff
@@ -0,0 +1,36 @@
+* = import Manifold.core
+* = import Manifold.digital
+
+Type TrafficLight = (
+	green:Boolean  = false,
+	yellow:Boolean = false,
+	red:Boolean    = false
+)
+
+GREEN_SECONDS  = 10
+YELLOW_SECONDS = 3
+TOTAL_SECONDS  = 2 * (GREEN_SECONDS + YELLOW_SECONDS)
+
+seconds = Manifold.Digital.count(hz: 1, mod: TOTAL_SECONDS)
+
+if (seconds < GREEN_SECONDS) {
+  north_light = south_light = (green: true)
```

Comment:
Those are assigned a default value of `false` in the type definition on lines 4-8